tests/functional/adapter/store_failures/fixtures.py

```diff
@@ -0,0 +1,42 @@
+seed_csv = """
+id,value
+1,1
+2,2
+3,3
+4,4
+""".lstrip()
+
+table_model = """
+{{config(materialized = "table")}}
```

Comment:
Table materialization is already configured in `project_config_update`.

Reply:
done